README.md

```diff
@@ -1,38 +1,88 @@
-Role Name
+Azure.azure_preview_modules
 =========
 
-A brief description of the role goes here.
+The role includes the most compelete and latest Azure modules for Ansible users. The update and bug fix are done in a more timely mannaer than official Ansible release. 
```

Comment:
"The role" -> "This tole"
"compelete" -> "complete"
"This role is the most complete and includes all the latest Azure modules"
"mannaer" -> "manner"